tools/cd_scripts/e2e_test.sh

```diff
@@ -198,15 +199,20 @@ function run_parallel_tests() {
   local -n test_array=$1
   local BUCKET_NAME=$2
   local pids=()
+  local benchmark_flags=""
+
   for test_dir_p in "${test_array[@]}"
   do
+    if [ $test_dir_p == "benchmarking" ]; then
+        benchmark_flags="-bench=."
```

Comment:
Please write comment above why this variable is require.

Reply:
added.